sql/moz-fx-data-shared-prod/app_store_syndicate/dataset_metadata.yaml

```
@@ -0,0 +1,12 @@
+friendly_name: app_store_syndicate
+description: |
+  Syndicated App Store data from Fivetran. User-facing views into this data can be found
+  in the ads dataset
+dataset_base_acl: syndicate
+user_facing: false
+syndication:
+  prod:
+    syndicated_project: "moz-fx-data-bq-fivetran"
+    syndicated_dataset: "app_store"
```

The syndicated dataset is `moz-fx-data-bq-fivetran.firefox_app_store`, and the syndicate dataset you're creating is `moz-fx-data-shared-prod.app_store_syndicate`.

```suggestion
    syndicated_dataset: "firefox_app_store"
```